Gemfile

```diff
@@ -3,11 +3,3 @@ source 'https://rubygems.org'
 gemspec
 
 gem 'ruby-fifo', github: 'shurizzle/ruby-fifo'
-
-group :development do
-  gem 'rspec'
```

Comment:
well the I believe pry, rubocop, and rspec can be moved back to Gemfile, in order to not waste gemspec =)